stan/math/prim/meta/max_size.hpp

```diff
@@ -1,44 +1,45 @@
 #ifndef STAN_MATH_PRIM_META_MAX_SIZE_HPP
 #define STAN_MATH_PRIM_META_MAX_SIZE_HPP
 
-#include <stan/math/prim/meta/length.hpp>
+#include <stan/math/prim/meta/size.hpp>
 
 namespace stan {
+namespace math {
```

Comment:
Is there a technical reason why you put it inside `stan::math`? I thought that the discussion in https://github.com/stan-dev/math/issues/1122#issuecomment-468953210 wasn't decided yet.

Reply:
Because size() and max_size are technically functions and should be moved to /fun. Also size() in the Stan languages currently uses the size(std::vector) and will also be extended soon for other types https://github.com/stan-dev/stanc3/issues/402 Not sure but I think all C++ functions exposed to compiler are in `stan::math`